.travis.yml

```diff
@@ -20,7 +20,7 @@ matrix:
       env: SYMFONY_VERSION='2.7.*'
     - php: 5.6
       env: SYMFONY_VERSION='2.8.*'
-    - php: 7.1
+    - php: 7.2
       env: SYMFONY_VERSION='4.0.*@dev'
```

Comment:
the one which should be tested explicitly here is the 3.4 LTS, like done for 2.7 and 2.8. The `4.0.*@dev` is useless